.gitignore

```diff
@@ -1,8 +1,5 @@
-phpunit.xml
-/nbproject/
-/coverage/
-/vendor/
-/docs/_build
+/doc/_build
+/vendor
 composer.lock
-composer.phar
+phpunit.xml
 .php_cs.cache
```

Comment:
Please sort the files as well like in https://github.com/sonata-project/SonataUserBundle/blob/3.x/.gitignore